migration-engine/ARCHITECTURE.md

```diff
@@ -39,8 +39,8 @@ There are a lot of assumptions that need to hold for a down migration to "just w
   by the bad deployment.
 - The down migration actually works. Are your down migrations tested?
 
-In short, down migrations give you a sense of security, but it is a false sense
-of security.
+In short, down migrations give you a sense of security, but it is often a false
```

Comment:
Oh so opinionated 😆

Reply:
Migrate itself is a very opinionated tool — it's also incomplete, but hiding the opinionation makes it seem even more incomplete imo.